.github/ISSUE_TEMPLATE/security_issue.yml

```
@@ -3,7 +3,7 @@ name: "🛡️ Security Issue"
 description: |
   Notify Louis Lam about a security concern. Please do NOT include any sensitive details in this issue.
 # title: "Security Issue"
-labels: [security]
+labels: ["security", "P1: high"]
```

not sure about the labels.. they at least need to not contain `:` or ` `. Otherwise githubs UI is reall akward in searching with them.

They somewhat contain the same info as the other things, no sure if this adds value.
We should reduce some of the labels or add them more consistently in any case.

```suggestion
labels: ["security", "p1-high"]
```

---

yes, using those would be appreciated